templates/nginx.conf.j2

```diff
@@ -29,7 +29,7 @@ http {
 
     client_max_body_size {{ nginx_client_max_body_size }};
 
-    log_format  main  {{ nginx_log_format|indent(23) }};
+    log_format  main  {{ nginx_log_format|indent(23) }}
```

Comment:
Should this semi-colon be re-added?

Reply:
Hello Colin, thanks for pointing this out. I don't know why this would have been edited from the change I've proposed. Maybe I forked the repo with the semi-colon missing, it was added after and now seen as a diff here.